proto/xudrpc.proto

```diff
@@ -220,7 +220,8 @@ message CancelOrderRequest {
 }
 
 message CancelOrderResponse {
-  bool result = 1 [json_name = "result"];
+  // True means that the order was canceled and false means that it wasn't
```

Comment:
So I know this comment is essentially verbatim what I said in my last comment here, but I wasn't thinking about the actual wording at the time and when I read it here it doesn't mesh with the way other comments are worded. I'd prefer "Indicates whether an order was successfully canceled" if that's alright.